lib/apkstats/plugin.rb

```diff
@@ -137,6 +145,22 @@ def compare_with(other_apk_filepath, do_report: true)
           md << "Download Size Change | #{size.to_s_b} Bytes. (#{size.to_s_kb} KB) " << "\n"
         end
 
+        result[:base][:reference_count].tap do |reference_count|
+          md << "New Reference Count | #{reference_count}" << "\n"
+        end
+
+        diff[:reference_count].tap do |reference_count|
+          md << "Reference Count Change | #{reference_count}" << "\n"
+        end
+
+        result[:base][:dex_count].tap do |dex_count|
+          md << "New Number of dex file(s) | #{dex_count}" << "\n"
+        end
+
+        diff[:dex_count].tap do |dex_count|
+          md << "Number of dex file(s) Change | #{dex_count}" << "\n"
+        end
```

Comment:
```
lib/apkstats/plugin.rb:99:5: C: Metrics/MethodLength: Method has too many lines. [63/60]
    def compare_with(other_apk_filepath, do_report: true) ...
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Fixing this rule is difficult for me.
I'm not very familiar with ruby, so I just obey code style.
Do you know to solve this warning? @jmatsu

Reply:
Good question! 

We can disable rubocop by inline comments as like IntelliJ's inspection.

*To disable Metrics/MethodLength rule*

```

# rubocop:disable Metrics/MethodLength

# <method comments>
def ...
```

*To re-enable Metrics/MethodLength rule*

```
...
end

# rubocop:enable Metrics/MethodLength
```

I'm also thinking it's hard to make this method smaller, actually.
`Metrics/AbcSize` is already disabled as for this method, so could you please follow it?
https://github.com/jmatsu/danger-apkstats/blob/8d09b6f151c970bb90ae7d51bdb8dc471f0b36ea/lib/apkstats/plugin.rb#L92

Reply:
Thanks for your suggestion!
I'll try rubocop:disable and enable comment.